pkgs/development/python-modules/dbt-common/default.nix

```diff
@@ -69,21 +58,11 @@ buildPythonPackage rec {
     typing-extensions
   ] ++ mashumaro.optional-dependencies.msgpack;
 
-  nativeCheckInputs = [
-    pytestCheckHook
-    pytest-xdist
-    pytest-mock
-  ];
-
-  disabledTests = [
-    # Assertion errors (TODO: Notify upstream)
-    "test_create_print_json"
-    "test_events"
-    "test_extra_dict_on_event"
-  ];
-
   pythonImportsCheck = [ "dbt_common" ];
 
+  # No tests in the pypi archive
+  doCheck = false;
```

Comment:
I think keeping the nativeCheckInputs and disabledTests even if doCheck is false is valuable, since it does encode relevant domain info for future contributors

Reply:
Notably, the nativeCheckInputs are not a part of the build closure when doCheck=false

Reply:
I don't consider this blocking, the rest LGTM